src/Data/ProgrammesDb/EntityRepository/CoreEntityRepository.php

```diff
@@ -203,6 +203,54 @@ public function findByIds(array $ids): array
             ->getQuery()->getResult(Query::HYDRATE_ARRAY);
     }
 
+    public function findDescendantsByType(array $ancestryDbIds, string $entityType, ?int $limit, int $offset) : array
+    {
+        $qb = $this->getEntityManager()->createQueryBuilder()
```

Comment:
Do we want to make sure that the $entityType isn't a Clip or Episode? @BjornTwachtmann